tldw_Server_API/app/core/LLM_Calls/LLM_API_Calls.py

```
@@ -62,11 +63,133 @@
+            raise err
+        return None
+
+
+class _SessionShim:
+    def __init__(
+        self,
+        *,
+        total: int = 3,
+        backoff_factor: float = 1.0,
+        status_forcelist: Optional[list[int]] = None,
+        allowed_methods: Optional[list[str]] = None,
```

Ruff (0.14.3)

Unused method argument: `allowed_methods`

(ARG002)